clients/fides-js/src/lib/i18n/i18n-utils.ts

```diff
@@ -36,6 +37,63 @@ export function areLocalesEqual(a: Locale, b: Locale): boolean {
   );
 }
 
+function matchLocale(
```

Comment:
I don't think you should need to extract this function (see other comments!).

If you _do_ need to extract it though, it still needs some kind of docs comment to explain why - even just a "see `matchAvailableLocales` for details" or similar

Reply:
yup you're right, now that we can simplify the matching 👍